modules/apps/app-builder/app-builder-service/service.xml

```diff
@@ -49,6 +48,28 @@
 
 		<reference entity="User" package-path="com.liferay.portal" />
 	</entity>
+	<entity local-service="true" name="AppBuilderAppDeployment" remote-service="false">
+
+		<!-- PK fields -->
+
+		<column name="appBuilderAppDeploymentId" primary="true" type="long" />
+
+		<!-- Other fields -->
+
+		<column name="appBuilderAppId" type="long" />
+		<column name="deploymentType" type="String" />
+		<column name="settings" type="String" />
+
+		<!-- Finder methods -->
+
+		<finder name="AppBuilderAppId" return-type="Collection">
+			<finder-column name="appBuilderAppId" />
+		</finder>
+		<finder name="A_D" return-type="AppBuilderAppDeployment">
```

Comment:
should be fetchByAppBuilderAppDeploymentType?

Reply:
No. Finders in service.xml when use more that one column as filter are named with the first letter of each filter column.